### ISSUE TITLE
JMX monitoring for cluster status

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/terrastore#48]
```
Implement basic JMX capabilities for monitoring the following
cluster-related information:
- Ensemble configured clusters.
- Cluster name and availability status.
- Available nodes per cluster.
- Node name and host.
```

Original issue reported on code.google.com by `sergio.b...@gmail.com` on 17 Apr 2010 at 11:31
- Blocked on: #47

**Comment by GoogleCodeExporter** [comment]
```
Implemented.
```

Original comment by `sergio.b...@gmail.com` on 2 May 2010 at 5:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 2 May 2010 at 5:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 9 has been merged into this issue.
```

Original comment by `sergio.b...@gmail.com` on 2 May 2010 at 5:41

**Comment by GoogleCodeExporter** [comment]
```
JMX monitoring has been removed in favor of REST stats, see issue 91.
```

Original comment by `sergio.b...@gmail.com` on 11 Sep 2010 at 7:55
